Training Plan/BeginningTraining.md

```diff
@@ -0,0 +1,133 @@
+# Basic training Schedule
+
+## First Meeting:
+ - [ ] Install software
+  - [WPI VS Code](https://github.com/wpilibsuite/allwpilib/releases)
+  - FRC Update (Driver Station)
+  - [GitKraken](https://www.gitkraken.com/)
+  - [Git](https://git-scm.com/)
+  - [Radio programming tool](https://docs.wpilib.org/en/latest/docs/software/getting-started/radio-programming.html) (Advanced Students)
+ - [ ] Demo of basic driving robot code
+  - Start with OI and subsystem created, including default command defined
+  - Talk through what is required as input to drive a robot
+  - Let students drive the robot
+
+## Second Meeting:
+ - [ ] Comments
+  - Explain comments
+  - Add single line comment inside drive command
+  - Add multi-line function comment for drive command
+ - [ ] Variables
+  - int
+    - Whole numbers, integars are passed to create motor controllers
+  - float/double
```

Comment:
Do we even use floats anywhere? I mean we might as well talk about them, but a quick "These are like doubles but smaller and less precise"